.devcontainer/host_setup.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 # pull base image
-docker pull ghcr.io/commaai/openpilot-base:latest
```

Comment:
it might make since to rebuild the base image here instead of pulling, so you can test local changes to the base image. It should be about the same amount of time, since it will use the remote image as cache